common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3748,6 +3748,12 @@
     },
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_ENCODER" : {
+    "message" : [
+      "Found unsupported encoder. Try switching to an expression encoder. (For more information on Encoders, consult: https://spark.apache.org/docs/latest/api/java/index.html?org/apache/spark/sql/Encoder.html)"
```

Comment:
Could you follow other items in the file for doc refs, see
https://github.com/apache/spark/blob/0d938de112dfe142f40d4c6f86cfa1e6e32210ec/common/utils/src/main/resources/error/error-classes.json#L1523

Reply:
Generally, I understood that these are generated files that are being referenced, but I referenced the file with all error state support and then I introduced a link to Encoder doc there.